webpack/webpack.common.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const srcScripts = '../src/scripts/';
```

Comment:
If you're going to use `path.join`, why are you manually joining these parts?

Reply:
The entries or the imports? I have change the entries to not manually join using `+`.

Reply:
(This variable is a bunch of manually-joined paths)

Reply:
Got it. Fixed.